LandMark/Feature/Home/OtherPage/OtherPageViewModel.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  OtherPageViewModel.swift
+//  LandMark
+//
+//  Created by Lucas Migge on 30/10/24.
+//
+
+import SwiftUI
+
+extension OtherPageView {
+    class ViewModel: ObservableObject {
+        
+        let coordinator: HomeCoordinator
```

Comment:
Mesma coisa sobre referência cíclica que o comentário acima.
```suggestion
        private(let) weak var coordinator: HomeCoordinator
```